srcpkgs/clickclack/patches/support-cross-compilation.patch

```diff
@@ -0,0 +1,22 @@
+ Makefile | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index b593b7f..1bf7a8e 100644
+--- a/Makefile
++++ b/Makefile
+@@ -2,12 +2,13 @@ PREFIX:=/usr
+ 
+ PROGRAMS = clickclack
+ 
++CC ?= $(CROSS_COMPILE)gcc
```

Comment:
$CC is already set by make and we override it, so I don't think we have to patch this

Reply:
Yeah technically this line addition isn't needed, I suppose I can drop it as it seems the `CROSS_COMPILE` part isn't useful anyway as `CC=cc` is set by default anyway and should be sufficient; I'll send a v2 and the new patch can be used here as well.

Reply:
[v2 now submitted](https://lists.sr.ht/~mil/sxmo-devel/patches/32157), feel free to grab the simpler first patch from there now @Eloitor

Reply:
Thank you